pkg/account/api/admin_account.go

```diff
@@ -944,12 +945,35 @@ func (s *AccountService) GetMyOrganizations(
 		}
 		return nil, dt.Err()
 	}
-	myOrgs := make([]*accountproto.MyOrganization, 0, len(accountsWithOrg))
+	if s.containsSystemAdminOrganization(accountsWithOrg) {
+		resp, err := s.environmentClient.ListOrganizations(
+			ctx,
+			&environmentproto.ListOrganizationsRequest{
+				Disabled: wrapperspb.Bool(false),
+				Archived: wrapperspb.Bool(false),
+			})
+		if err != nil {
+			return nil, err
```

Comment:
We must localize the error message before returning it.
Can you add a unit test for this case, too?

Reply:
@cre8ivejp 
Thank you, I fixed it!